### PR TITLE
Fix login e2e tests

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

https://github.com/medic/angular10-migration/issues/161
https://github.com/medic/angular10-migration/issues/164
https://github.com/medic/angular10-migration/issues/140
https://github.com/medic/angular10-migration/issues/165
https://github.com/medic/angular10-migration/issues/147

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: tests/e2e/login/privacy-policy.spec.js [hunk 1]
```diff
@@ -126,7 +127,13 @@ describe('Privacy policy', () => {
   });
 
   describe('for a french offline user', () => {
+    let passed = false;
     afterEach(async () => {
+      if (!passed) {
+        // I suspect this test is failing because of a conflict.
```
Comment (dianabarsan) [inline tests/e2e/login/privacy-policy.spec.js:133]: Added issue: https://github.com/medic/cht-core/issues/6960

**Comment by dianabarsan** [comment]
hello @newtewt 
Can you, pretty please, have a look at this? Thanks for your time!

File: tests/e2e/login/privacy-policy.spec.js [hunk 2]
```diff
@@ -126,7 +127,13 @@ describe('Privacy policy', () => {
...
+        const userDoc = await utils.requestOnTestDb('/org.couchdb.user:offline?conflicts=true');
+        console.log(JSON.stringify(userDoc, null, 2));
```
Comment (newtewt) [inline tests/e2e/login/privacy-policy.spec.js:135]: Wondering if there needs to be some clarification in the log incase someone looks. We have a decent amount of noise.

File: tests/e2e/login/privacy-policy.spec.js (hunk 1 above)
Comment (newtewt) [reply tests/e2e/login/privacy-policy.spec.js:133]: I keep running into things that protractor/jasmine do not does out of the box that other frameworks I've used do. Like afterEach having deails about the spec that ran. IE pass/fail. Nothing to change just a rant.

**Review by newtewt — APPROVED** [review]
Only one question.

File: tests/e2e/login/privacy-policy.spec.js (hunk 2 above)
Comment (dianabarsan) [reply tests/e2e/login/privacy-policy.spec.js:136]: Done.

File: tests/e2e/login/privacy-policy.spec.js (hunk 1 above)
Comment (dianabarsan) [reply tests/e2e/login/privacy-policy.spec.js:133]: yea, 100% agree. 
The framework does use this info though. If a beforeEach or afterEach fails, they appear as fails attached to the test they ran for.

**Pull request merged**